Wabbajack/View Models/Gallery/ModListGalleryVM.cs

```diff
@@ -173,11 +196,20 @@ public override void Unload()
         private List<string> GetGameTypeEntries()
         {
             List<string> gameEntries = new List<string> { ALL_GAME_TYPE };
-            foreach (var gameType in EnumExtensions.GetAllItems<Game>() )
-            {
-                gameEntries.Add(EnumExtensions.GetDescription<Game>(gameType));
-            }
+            gameEntries.AddRange(EnumExtensions.GetAllItems<Game>().Select(gameType => gameType.GetDescription<Game>()));
             return gameEntries;
         }
+
+        private void UpdateFiltersSettings()
+        {
+            if (!settings.IsPersistent)
```

Comment:
I think there is a subtle edge case here.  Consider that the user has had IsPersistent on, and has their filters saved.  Now they start WJ up again, and decide they don't like that.  They turn IsPersistent off.  Now when this function gets called to save the settings, we short circuit and return.

Well, the end result is that his previously saved filter settings are just left as-is, so the next time they start up the filters won't be clean, they'll have the stale "last saved" filters.

This call needs to actively wipe the filter settings clean if IsPersistent is off